tests/test_context_diagrams.py

```diff
@@ -148,3 +149,14 @@ def test_context_diagram_pass_params_to_render(
 
         with pytest.raises(KeyError):
             without_hierarchy[uuid]  # pylint: disable=pointless-statement
+
+
+def test_context_diagram_with_derived_interfaces(
+    model: capellambse.MelodyModel,
+) -> None:
+    obj = model.by_uuid(TEST_DERIVED_UUID)
+
+    diag = obj.context_diagram
+    diagram = diag.render(None, display_derived_interfaces=True)
```

Comment:
```suggestion
    diagram = obj.context_diagram.render(None, display_derived_interfaces=True)
```
nit: IMO, the names `diag` and `diagram` are a bit confusing here, since to me the first is simply the short form of diagram. So I would either rename the first to `ctx_diagram` or `ctx_diag` and the second to `rendered_diag` or `rendered_diagram`, or just apply my suggestion and remove the additional variable here.